### PR TITLE
feat: allow multiple series in same analysis

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

File: src/global.d.ts [hunk 1]
```diff
@@ -1 +1,24 @@
-declare module 'common-spectrum';
+declare module 'common-spectrum' {
```
Comment (lpatiny) [inline src/global.d.ts:1]: Please create a PR in common-spectrum so that you can use it everywhere.

File: src/from/fromMulChannelCap.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ function keyMapper(keys: string[]) {
   });
 }
 
-export function fromMulChannelCap(text: string) {
+export function fromMulChannelCap(text: string, analysis?: Analysis) {
```
Comment (lpatiny) [inline src/from/fromMulChannelCap.ts:29]: The second argument could be options={} and one of the options (the only one currently) is analysis.

**Review by lpatiny — COMMENTED** [review]
Good idea. I think we should do the same in common-spectrum in 'fromJcamp'

File: src/global.d.ts (hunk 1 above)
Comment (maasencioh) [reply src/global.d.ts:1]: I'll use it for migration later, apparently I'll need to do it in any case 😫

File: src/from/fromMulChannelCap.ts (hunk 2 above)
Comment (maasencioh) [reply src/from/fromMulChannelCap.ts:29]: what other option could be added?

Comment (lpatiny) [reply src/from/fromMulChannelCap.ts:29]: Currently none. But by habit everything that is optional we put as a property of an option object. It also allows to add more options without breaking changes.

**Comment by maasencioh** [comment]
closed in favor of https://github.com/cheminfo/common-spectrum/pull/12

**Pull request closed without merging**